pkg/auth/arn.go

```diff
@@ -109,3 +111,20 @@ func ArnMatch(src, dst string) bool {
 	}
 	return false
 }
+
+func ParsePolicyResourceAsList(resource string) ([]string, error) {
+	trimmed := strings.TrimSpace(resource)
+	if len(trimmed) == 0 {
+		return []string{resource}, nil
```

Comment:
This should be error invalid arn. 
even if not, return empty array, `[]string{}` but - i prefer error invalid ARN at this level with error wrapping